docker-compose.desktop.yml

```diff
@@ -35,19 +39,17 @@ services:
 
         https:// {
           log
-          reverse_proxy * firezone:13000
-          tls internal {
-            on_demand
-          }
+          reverse_proxy * 172.25.0.100:${PHOENIX_PORT:-13000}
+          ${TLS_OPTS:-}
```

Comment:
We were ignoring the `TLS_OPTS` env on Mac/Win: https://github.com/firezone/firezone/blob/master/scripts/install.sh#L104.

Reply:
Hm yeah good catch! I want to say this may have been this way due to an assumption that Win/Mac could only run for local use cases, but that assumption may not always be valid.